tst/SdpApiTest.cpp

```diff
@@ -472,6 +472,8 @@ a=end-of-candidates
     }
 
     ASSERT_EQ(2, ssrcLines.size());
+
+    EXPECT_EQ(STATUS_SUCCESS, freePeerConnection(&pRtcPeerConnection));
```

Comment:
This is really concerning. Our instrumented stuff is really nice, but it can't catch the boundaries between libraries (we have had lots of leaks in OpenSSL usage)

I will investigate